### PR TITLE
Create home screen ui

### PR DESCRIPTION
**Pull request by shinnosuke14202**
## Related Tickets
- [#Task 78523: Content](https://edu-redmine.sun-asterisk.vn/issues/78523)
## WHAT
- Create Home Screen UI
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

**Review by nang2002hv — APPROVED** [review]

**Review by nhikiu — APPROVED** [review]

File: app/src/main/java/com/example/book/screen/MainActivity.kt [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
     }
 
     override fun initView() {
-        // todo
+        setNextFragment(HomeFragment.newInstance())
+        setNavigation()
+    }
+
+    private fun setNavigation() {
+        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottomNavigation)
```
Comment (ghost) [inline app/src/main/java/com/example/book/screen/MainActivity.kt]: Dùng viewBinding đi e zai

File: app/src/main/java/com/example/book/screen/home/HomeFragment.kt [hunk 2]
```diff
@@ -1,23 +1,92 @@
...
+    private val homeBookAdapter: HomeBooksAdapter by lazy { HomeBooksAdapter() }
+    private val homeAuthorAdapter: HomeAuthorsAdapter by lazy { HomeAuthorsAdapter() }
+
     override fun inflateViewBinding(inflater: LayoutInflater): FragmentHomeBinding {
         return FragmentHomeBinding.inflate(inflater)
     }
 
     override fun initView() {
-        // todo
+        Log.i("tag", "hello")
```
Comment (ghost) [inline app/src/main/java/com/example/book/screen/home/HomeFragment.kt]: tạo string TAG để là constant em nha

File: app/src/main/java/com/example/book/screen/home/HomeFragment.kt [hunk 3]
```diff
@@ -1,23 +1,92 @@
...
+
+    override fun onError(exception: Exception) {
+        Toast.makeText(context, exception.message, Toast.LENGTH_SHORT).show()
+    }
+
+    private fun authorsData(): List<Author> {
+        val authors =
+            listOf(
+                Author(
+                    name = "J. K. Rowling",
```
Comment (ghost) [inline app/src/main/java/com/example/book/screen/home/HomeFragment.kt]: Sao lại hard code thế này em?

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -6,13 +6,31 @@
...
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@+id/glBottom"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <com.google.android.material.bottomnavigation.BottomNavigationView
+        android:id="@+id/bottomNavigation"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```
Comment (ghost) [inline app/src/main/res/layout/activity_main.xml]: Đưa 0dp vào file dimen

**Review by ghost — CHANGES_REQUESTED** [review]
anh có cmt nèe

File: app/src/main/java/com/example/book/screen/home/HomeFragment.kt (hunk 3 above)
Comment (shinnosuke14202) [reply app/src/main/java/com/example/book/screen/home/HomeFragment.kt]: Không có API trả về danh sách tác giả nên em để thế để sau bấm vô tác giả sẽ hiển thị sách của tác giả đó ạ.

**Review by ghost — APPROVED** [review]

**Pull request merged**